e2e-tests/constants/application.js

```diff
@@ -6,6 +6,10 @@ export const APPLICATION = {
   POLICY_TYPE: {
     SINGLE: 'Single contract policy',
     MULTIPLE: 'Multiple contract policy',
+    ABBREVIATED: {
+      SINGLE: 'Single contract policy',
```

Comment:
should these be the same value as above?

Reply:
Whoops, no - fixed. Good spot